tf-modules/cloudtrail/variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "name_prefix" {
+  description = "name to be prepended to all resources"
+}
+
+variable "aws_cloud" {
+  description = "set to 'aws-us-gov' if using GovCloud, otherwise leave the default"
+  default     = "aws"
+}
+
+variable "kms_key_id" {
+  description = "KMS key ARN for encryption of logs"
+  default     = ""
+}
+
+variable "include_global_service_events" {
+  description = "boolean, if true logs IAM events as well as region-specific ones"
+  default     = true
+}
+
+variable "enable_logging" {
+  description = "boolean, defines if logging should be started or stopped"
```

Comment:
I'm not sure what "logging" this is referring to, maybe either explain that, or go more towards: "Boolean, maps to `aws_cloudtrail`'s `enable_logging` parameter"

Reply:
Agreed. I did spend a while looking at the screen when I wrote this one. Your solution is straightforward, and I added it in the commit below 👇